src/app/shared/main-navigation/mat-main-nav/mobile-menu/mobile-menu.component.ts

```diff
@@ -11,7 +11,12 @@ import { environment } from "@environment";
 export class MobileMenuComponent implements OnInit {
   menuItems$: IMenuItem[] = menuItems;
   hostURI: string = environment.adminsUIHostUri;
+  @Output() closeMenu = new EventEmitter();
   constructor() {}
 
   ngOnInit(): void {}
+
+  onMenuClick(): void {
```

Comment:
would be nice to have a unit test to cover this method

Reply:
will action